ci/nodepool/scripts/prepare_pulp3_node.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+HOSTNAME="$1"
+DOCKER="$(grep docker 2>/dev/null <<< ${HOSTNAME})"
+
+echo "Disable selinux"
+sudo setenforce 0
+
+# Fail immediately on error
+set -ex
+
+source bootstrap.sh
+
+sudo "${PKG_MGR}" install -y gcc git ansible libselinux-python
+
+git clone https://github.com/pulp/devel.git
+cd devel
+git checkout 3.0-dev
+sed -i -e 's/pulp_user: vagrant/pulp_user: jenkins/g' ansible/pulp-from-source.yml
```

Comment:
Instead of editing the playbook, pass `-e pulp_user=jenkins` when calling `ansible-playbook`.